src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,113 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private final int defaulCapacity = 16;
+    private int capasity;
+    private final float loadFactor = 0.75f;
+    private int size;
+    private Node<K,V>[] table;
+
+    public MyHashMap() {
+        capasity = defaulCapacity;
+        size = 0;
+        table = new Node[capasity];
+    }
+
+    static class Node<K,V> {
+        private final K key;
+        private V value;
+        private Node<K,V> next;
+
+        Node(K key, V value,Node<K,V> next) {
+            this.key = key;
+            this.value = value;
+            this.next = next;
+        }
+    }
 
     @Override
     public void put(K key, V value) {
+        if (size > table.length * loadFactor) {
+            resize();
+        }
+        Node<K,V> node = table[hash(key) % table.length];
```

Comment:
You are calculating the hash and performing modulus operation twice for the same key. It would be better to calculate it once and store it in a local variable to reuse it.